.github/workflows/_runner-determinator.yml

```diff
@@ -164,8 +171,10 @@ jobs:
 
           if __name__ == "__main__":
               main()
+
           EOF
-          cat get_workflow_type.py
+
+          cat runner_determinator.py
 
```

Comment:
```suggestion
      - name: guarantee `.github/scripts/runner_determinator.py` matches workflow
         run: cmp -s .github/scripts/runner_determinator.py get_workflow_type.py
```

Reply:
+1 I think this is a good idea!

Reply:
We can't have this comparison check because of the whole reason that we're embedding this file in the first place:

During action execution, the version that's in the yml file can be out of sync with the version that's checked into the repo, causing this action to fail for someone who hasn't rebased onto `main` yet